pkgs/applications/audio/pulseaudio-dlna/default.nix

```diff
@@ -14,24 +31,34 @@ assert flacSupport -> flac != null;
 assert soxSupport -> sox != null;
 assert vorbisSupport -> vorbis-tools != null;
 
-let
-  zeroconf = pythonPackages.callPackage ./zeroconf.nix { };
-in
-pythonPackages.buildPythonApplication {
+buildPythonApplication rec {
   pname = "pulseaudio-dlna";
-  version = "unstable-2017-11-01";
+  version = "unstable-2021-01-21";
 
   src = fetchFromGitHub {
-    owner = "masmu";
+    owner = "Cygn";
```

Comment:
I am not sure if we should use this fork.

Reply:
Addressed in 03edc17